Prevent configure script from assuming that if we don't want X11, then
we don't want OpenGL either.
--- configure.orig	2022-05-20 11:11:32.000000000 -0500
+++ configure	2022-06-02 15:55:01.000000000 -0500
@@ -20878,7 +20878,7 @@
 
 
 
-if test "$no_x" != "yes" -a "$no_gl" != "yes" \
+if test "$no_gl" != "yes" \
 	-a "$no_glu" != "yes" -a "$no_glut" != "yes" ; then
   HAVE_OPENGL=yes
 fi
